src/TLMessage/TLMessage/TLServerMessage.php

```diff
@@ -53,8 +52,8 @@ protected function throwIfIncorrectType(AnonymousMessage $anonymousMessage): voi
     {
         if(!static::isIt($anonymousMessage)) {
             $msg = $anonymousMessage->getType().' instead of '.get_called_class().' class';
-            if ($anonymousMessage instanceof RpcError) {
-                $msg .= ' with error '.$anonymousMessage->getErrorString();
+            if ($anonymousMessage->getType() === 'rpc_error') {
```

Comment:
Use isIT() method to verify type - in order to keep all the names in one place

Reply:
fixed